docs/deploy_narrative.md

```diff
@@ -1,64 +1,109 @@
 ## Narrative Development & Deployment HowTo
 
+This document describes how to release and deploy the Narrative Interface app onto the different KBase environments. It's intended for KBase developers and operators.
```

Comment:
"operators" seems a bit strange -- maybe sys admins?

Reply:
Yeah, that's the right word. I got hung up on "devops" but that didn't seem right.